src/lib/CRSF/CRSF.cpp

```diff
@@ -201,7 +201,7 @@ void ICACHE_RAM_ATTR CRSF::sendLinkStatisticsToTX()
 
 void ICACHE_RAM_ATTR CRSF::sendLUAresponse(uint8_t val[])
 {
-    uint8_t dataLength = sizeof(val);
+    uint8_t dataLength = 4;
```

Comment:
I need to dig deeper...  but is there any reason this should be a fixed value?

Reply:
Ah, I just reread you comment.  Yep lets exclude it and get it merged.

Reply:
I reverted the change. The sendLUAresponse method is allways called for 4 bytes and so I did assign this value to prevent the problem that sizeof calculates the size of the pointer variable and not the array in this case.